server/Mistakes.Journal.Api/Logic/Mistakes/Models/Label.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Mistakes.Journal.Api.Logic.Mistakes.Models
+{
+    public class Label
+    {
+        public Guid Id { get; }
+        public string Name { get; set; }
+        public string Color { get; set; }
+        public ICollection<MistakeLabel> MistakeLabels { get; set; }
```

Comment:
Is it required? Is there a case we use this collection that way? I mean, from label's side?